shared/engine/index.js

```diff
@@ -41,8 +41,9 @@ class EngineChannel {
     return yield Saga.takeFromChannelMap(this._map, key)
   }
 
-  *race(options: ?{timeout?: number, racers?: Object}): Generator<any, any, any> {
+  *race(options: ?{timeout?: number, removeNs: boolean, racers?: Object}): Generator<any, any, any> {
     const timeout = options && options.timeout
+    const removeNs = (options && options.removeNs) || false
```

Comment:
but to expand the confusing part a bit, it was changing what we were calling things without any explicit request. I thought having to pass in removeNs explicitly made it more obvious it was going to change those keys